aya/src/programs/cgroup_skb.rs

```diff
@@ -87,6 +88,7 @@ impl CgroupSkb {
         &mut self,
         cgroup: T,
         attach_type: CgroupSkbAttachType,
+        flag: CgroupAttachMode,
```

Comment:
here and everywhere else: please rename the argument name from flag to mode

Reply:
Right!  Done.